frontend/src/components/maps/leaflet/Map.tsx

```diff
@@ -28,19 +28,19 @@ type MapProps = {
   lat: number;
   lng: number;
   zoom: number;
-  parcels: IParcel[];
+  parcels: IProperty[];
```

Comment:
I wonder if it makes more sense to update the naming conventions here and elsewhere.  If it’s a `property` object then use `properties` not `parcels`.

Reply:
I thought about that and initially did it on my branch but then I saw Devin's PR and we will have **massive** conflicts which I'd like to avoid if possible.
We can file a tech-debt item to rename this across the frontend when the code is more stable after all the merges.